packages/salesforcedx-slds-linter/package.json

```diff
@@ -35,8 +36,11 @@
     "watch": "tsc -watch -p .",
     "clean": "shx rm -rf node_modules",
     "postinstall": "node ./node_modules/vscode/bin/install",
-    "test": "node ./node_modules/vscode/bin/test",
-    "test:unit": "node ./node_modules/vscode/bin/test"
+    "test": "npm run test:vscode-integration",
```

Comment:
Question - is the `test` script meant to run all the tests in each package? ie. in this one, should it be running the `vscode-integration` and `vscode-insiders-integration` scripts?

Reply:
@AnanyaJha That's a good point. I've been considering VS Code (and not Insiders) to be the default, but don't have a great reason for doing that. The only caveat I have for combining the two scripts is that before running `npm run test` locally, users will have to quit out of both VS Code and Insiders since the tests fail when another instance of the same editor is open in another window. This isn't really a problem, though, since the scripts to only run the test against VS Code or Insiders exist. 

I'll add `npm run test:vscode-insiders-integration` to the `test` scripts.

Reply:
We've decided to combine the `vscode-integration` and `vscode-insiders-integration` scripts under the `test` script, but not until this work (W-4889431) is done to break up our autobuild into multiple stages. Combining the scripts now would effectively add the vscode-insiders-integration tests to our current build, which would make the build time significantly longer.